src/cljs/main/broadfcui/common/input.cljs

```diff
@@ -25,6 +25,7 @@
    (fn [{:keys [state props]}]
      (style/create-text-field
       (merge {:ref "textfield"
+              :data-test-id (:data-test-id props)
```

Comment:
I didn't find this to be necessary because `props` gets merged in with this map (minus `:ref`, `:style`, and `:onChange`) and passed to `create-text-field`.